src/test/scala/com/expedia/www/haystack/commons/config/ConfigurationLoaderSpec.scala

```diff
@@ -17,33 +17,46 @@
 
 package com.expedia.www.haystack.commons.config
 
-import com.expedia.www.haystack.commons.unit.UnitTestSpec
+import org.scalatest.{FunSpec, Matchers}
 
-class ConfigurationLoaderSpec extends UnitTestSpec {
+class ConfigurationLoaderSpec extends FunSpec with Matchers {
 
-  "ConfigurationLoader.loadConfigFileWithEnvOverrides" should {
+  describe("configuration loader") {
+    val keyName = "traces.key.sequence"
 
-    "load a given config file as expected when no environment overrides are present" in {
-      Given("a sample HOCON conf file")
-      val file = "sample.conf"
```

Comment:
Can we add a test like this to prove it can load a file other than config/base.conf?

Reply:
I see. I think I didn't notice that you changed this file too.